migrations/1625802132075_my-first-migration.js

```diff
@@ -15,8 +15,8 @@ exports.up = pgm => {
       content VARCHAR(5000),
       created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
       last_edited TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
-      img_url VARCHAR(500),
-      video_url VARCHAR(500)
+      imageURL VARCHAR(500),
+      videoURL VARCHAR(500)
```

Comment:
It's important to stay consistent in how you name fields, there are fields now that have words separated by underscore and fields that are camelCased. In SQL it's pretty common to use underscores for variable names, because in SQL is usually not case sensitive as a standard (i.e. imageURL is the same is imageurl).